Copy editing (#200)
openforis/sepal-doc: docs/source/cookbook/optical_mosaic.rst

    :width: 49%
    :group: optical-mosaic-recipe

These overlay areas can be managed in various ways. For example, you can choose to keep only the raster data from the first or last dataset, combine the values of the overlay cells using a weighting algorithm, average the values of the overlay cells, or take the maximum or minimum value. In addition, certain corrections can be made to the image to account for clouds, snow, and other factors. These operations are complex and repetitive. SEPAL offers you an interactive and intuitive way to create mosaics in any area of interest (AOI).

.. Note::

>     You won't be able to retrieve the images if your SEPAL and Google Earth Engine (GEE) accounts are not connected. Navigate to `Connect SEPAL to GEE <../setup/gee.html>`__ to learn more.

Reviewer: ```suggestion
    You won't be able to retrieve the images if your SEPAL and Google Earth Engine (GEE) accounts are not connected. Navigate to :doc:`../setup/gee` to learn more.
```

an inline role that I discovered after writing this page. it makes sure that the title of the link is always the one from the linked page.

Using the main slider (2) define a season around the target date. This season defines two dates: a starting date and an ending date. SEPAL will then retrieve the mosaic images between those dates. 

The number of images in one single season of one year may not be enough to produce a correct mosaic. SEPAL provides two secondary sliders to increase the pool of images to create the mosaic. Both count the number of seasons SEPAL can retrieve in the past (:code:`Past season` - (3)) and in the future (:code:`Future season` - (4)). 

When the selection is done click on the :icon:`fa fa-check` :guilabel:`Apply` button.

.. thumbnail:: ../_images/cookbook/optical_mosaic/select_season.png
    :title: The season selection panel.
    :group: optical-mosaic-recipe

Sources
^^^^^^^

As mentioned in the introduction, a mosaic uses different raster datasets that can be obtained from multiple sources. SEPAL allows you to select data from multiple entry points. Below, you can find a description of these sources (click on the link to see the corresponding dataset information):

> -   :guilabel:`L8`: `Landsat 8 Tier 1 <https://developers.google.com/earth-engine/datasets/catalog/LANDSAT_LC08_C01_T1>`__. Landsat scenes with the highest available data quality are placed into Tier 1 and considered suitable for time-series processing analysis. Tier 1 includes Level-1 Precision Terrain (L1TP) processed data that have well-characterized radiometry and are inter-calibrated across the different Landsat sensors. The geo-registration of Tier 1 scenes will be consistent and within prescribed tolerances (<=12 m root mean square error [RMSE]). All Tier 1 Landsat data can be considered consistent and inter-calibrated (regardless of the sensor used) across the full collection.

Reviewer: this text is actually copy/pasted from the actual satellite documentation, even though it's not perfect, it should not be changed. after links that starts with "developers.google.com, I usually simply copy/pastde the doc, you can discard these lines

Snow masking
""""""""""""

Define how snowy pixels will be masked.

-   :guilabel:`on`: Masks snow. This tends to leave some pixels with shadowy snow.
-   :guilabel:`off`: Doesn't mask snow. Note that some clouds might get misclassified as snow, and because of this, disabling snow masking might lead to cloud artifacts.

Composing method
""""""""""""""""

After filtering the stack of pixels, SEPAL will compute the median value on the different bands of the image. The composing method will define how the final pixel value is extracted. 

-   :guilabel:`Medoid`: Uses the closest pixel from the median value. As a real pixel from the stack, the final value will embed metadata (e.g. the date of observation).
-   :guilabel:`Median`: Uses the computed value of the median. If no pixel is matching this value, the pixel will not embed any metadata. It tends to produce smoother mosaics. 

Analysis
--------

After selecting the parameters, you can start interacting with the scenes and begin the analysis.
In the upper-right corner, three tabs are available. They will allow you to customize the mosaic scene selection and export the final result.

> -   :icon:`fas fa-magic`: auto-select scenes

Reviewer: ```suggestion
-   :btn:`<fas fa-magic>`: auto-select scenes
```

> -   :icon:`fas fa-trash`: clear selected scenes

Reviewer: ```suggestion
-   :btn:`<fas fa-trash>`: clear selected scenes
```

> -   :icon:`fas fa-cloud-download-alt`: retrieve mosaic

Reviewer: ```suggestion
-   :btn:`<fas fa-cloud-download-alt>`: retrieve mosaic
```
